identity/src/main/java/com/android/identity/credentialtype/CredentialAttributeType.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.android.identity.credentialtype
+
+/**
+ * Attributes in credentials can have different types and this enumeration contains a type-system
+ * generic enough to be used across various credential formats. This is useful for wallet and reader
+ * user interfaces which wants to provide UI for inputting our displaying credentials attributes.
```

Comment:
Nit: "or" instead of "our" in the last sentence. Also missing initial short sentence (see other comment on `CredentialTypeRepository`.

Reply:
Done